### PR TITLE
Changes to the staff profiles

### PR DESCRIPTION
**Pull request by mngau**
Tidied up some profiles

### CODE REVIEW

File: _data/staff.yml [hunk 1]
```diff
@@ -31,7 +31,19 @@
   avatar: images/staff/veronica_finarelli.jpg
   #  socials:
   #  - icon_class: fa fa-linkedin
-  #    url: https://au.linkedin.com/in/hannahbrowne
+  #    url:
+
+- name: Richard Busso
+  position: Client Solutions Executive
+  desc:
+    Richard Busso is a connector within the industry that builds strong long lasting relationships and is the Client Solutions Executive at Cevo.
```
Comment (nonspecialist) [inline _data/staff.yml]: "who" builds, not "that" builds -- he is a who, not a what

File: _data/staff.yml [hunk 2]
```diff
@@ -31,7 +31,19 @@
...
+    Richard is a strategic leader who loves collaborating with businesses and people to solve complex problems.  His business acumen, enterprise knowledge along with his passion for connecting people makes him smile everyday.
```
Comment (nonspecialist) [inline _data/staff.yml]: rewrite: His business acumen and enterprise knowledge, along with his passion for connecting people makes him smile every day.

**Comment by cevopeter** [comment]
Thanks 
Happy for it to change.

thanks for reviewing it
rich

**Comment by mngau** [comment]
Grammatical changes have been made as per @nonspecialist's recommendations.

**Pull request merged**